compiler/generator/rust/rust_code_container.cpp

```diff
@@ -549,20 +601,20 @@ void RustCodeContainer::generateComputeHeader(int n, std::ostream* fOut,int fNum
     // Compute "compute" declaration
     tab(n, *fOut);
     tab(n, *fOut);
-    *fOut << "fn compute_arrays("
+    *fOut << "pub fn compute_arrays("
```

Comment:
I find it a bit strange that in the generated code the current order of methods is:

- `compute_arrays` comes first.
- then comes `new`, and all other "basic" methods, followed by UI related methods.
- the last method is `compute`.

Since `compute` and `compute_arrays` have a direct relationship, it would be nice to generate them next to each other. Could we just move these generating lines down into `generateComputeInterfaceHeader`?